ament_clang_format/cmake/ament_clang_format_lint_hook.cmake

```diff
@@ -0,0 +1,14 @@
+file(GLOB_RECURSE _source_files FOLLOW_SYMLINKS
+  "*.c"
+  "*.cc"
+  "*.cpp"
+  "*.cxx"
+  "*.h"
+  "*.hh"
+  "*.hpp"
+  "*.hxx"
+)
```

Comment:
What direct is this glob done in? Should these be prefixed with `CMAKE_CURRENT_SOURCE_DIR`?